app/Individual.php

```diff
@@ -446,7 +446,7 @@ public function getBirthPlace() {
 			}
 		}
 
-		return '';
+		return new Place(null, $this->tree);
```

Comment:
The constructor takes a string as its first argument, so perhaps: 

`return new Place('', $this->tree);a